robottelo/virtwho_utils.py

```diff
@@ -31,15 +34,15 @@ def get_system(system_type):
     """Return a dict account for ssh connect.
 
     :param str system_type: The type of the system, should be one of
-        ('satellite', 'guest').
+        ('satellite', 'esx', 'xen', 'hyperv', 'rhevm', 'libvirt', 'kubevirt').
     :raises: VirtWhoError: If wrong ``system_type`` specified.
     """
-    if system_type == 'guest':
+    if system_type in ['esx', 'xen', 'hyperv', 'rhevm', 'libvirt', 'kubevirt']:
         return {
-            'hostname': settings.virtwho.guest,
-            'username': settings.virtwho.guest_username,
-            'password': settings.virtwho.guest_password,
-            'port': settings.virtwho.guest_port,
+            'hostname': eval(f'virtwho.{system_type}.guest'),
+            'username': eval(f'virtwho.{system_type}.guest_username'),
+            'password': eval(f'virtwho.{system_type}.guest_password'),
+            'port': eval(f'virtwho.{system_type}.guest_port'),
```

Comment:
`eval` is usually highly discouraged. You can use `getattr` here instead:

```
'hostname': getattr(virtwho, system_type).guest,
```

Reply:
OK, will update this with the next commit.

Reply:
and is it because `it is slow, it is prone to bugs and strange behavior, and it makes the code hard to handle and very difficult to understand. `？

Reply:
@hkx303 all of these reasons, plus eval is potentially dangerous and might be root cause of security breach.
Basically, it's best to avoid eval unless you absolutely have to. And here you don't have to.